### PR TITLE
update dropbox authorization

### PR DESCRIPTION
**Pull request by gorzalczany**
Fixes https://github.com/Deseteral/lebkuchen-fm/issues/200

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,16 +60,26 @@ This projects consists of these modules:
 Core LebkuchenFM Node.js service with MongoDB storage that communicates with clients over WebSockets and REST endpoints.
 
 #### Configuration
+Configure your instance via Environment variables.
```
Comment (Deseteral) [inline README.md:63]:
```suggestion
Configure your instance via environment variables.

```

File: README.md [hunk 2]
```diff
@@ -60,16 +60,26 @@ This projects consists of these modules:
...
+##### Service
+- `MONGODB_URI` - MongoDB connection string
+- `PORT` - port on which the service will be running (automatically injected by cloud providers)
+- `LOCALE` - language of the service
 - `COMMAND_PROMPT` - command prompt (optional, defaults to `/fm`)
+
+##### Youtube player
+- `YOUTUBE_API_KEY` - YouTube Data API token
+##### Discord
```
Comment (Deseteral) [inline README.md:72]:
```suggestion

##### Discord
```

File: README.md [hunk 3]
```diff
@@ -60,16 +60,26 @@ This projects consists of these modules:
...
 - `DISCORD_CHANNEL_ID` - ID of the Discord channel where the bot is allowed to run
 - `DISCORD_CLIENT_ID` - Discord application ID
 - `DISCORD_GUILD_ID` - ID of the Discord guild (server) where the bot will operate
 - `DISCORD_TOKEN` - token of the Discord bot
-- `DROPBOX_TOKEN` - Dropbox API token used for persisting files
-- `LOCALE` - language of the service
-- `MONGODB_URI` - MongoDB connection string
-- `PORT` - port on which the service will be running (automatically injected by cloud providers)
-- `YOUTUBE_API_KEY` - YouTube Data API token
+##### Dropbox - used for persisting files
```
Comment (Deseteral) [inline README.md:77]:
```suggestion

##### Dropbox - used for persisting files
```

File: README.md [hunk 4]
```diff
@@ -60,16 +60,26 @@ This projects consists of these modules:
...
+- `DROPBOX_CLIENT_ID` - Dropbox App Key 
+- `DROPBOX_SECRET` - Dropbox App Secret 
+- `DROPBOX_REFRESH_TOKEN` - Dropbox [refresh token][how_to_get_refresh_token] used for persisting files
+
+[how_to_get_refresh_token]: https://www.codemzy.com/blog/dropbox-long-lived-access-refresh-token#how-can-i-get-a-refresh-token-manually
```
Comment (Deseteral) [inline README.md:82]: Why link to some random post? Is there no official Dropbox documentation for this stuff?

File: packages/service/src/infrastructure/configuration.ts [hunk 5]
```diff
@@ -44,7 +50,9 @@ class Configuration {
       process.env.DISCORD_CLIENT_ID || '',
       process.env.DISCORD_GUILD_ID || '',
       process.env.DISCORD_TOKEN || '',
-      process.env.DROPBOX_TOKEN || '',
+      process.env.DROPBOX_CLIENT_ID || '',
+      process.env.DROPBOX_SECRET || '',
+      process.env.DROPBOX_REFRESH_TOKEN || '',
```
Comment (Deseteral) [inline packages/service/src/infrastructure/configuration.ts:55]: I'm starting to feel like it's dumb to default to empty string when there is no value, but this is outside the scope of this PR.

File: package-lock.json
Comment (Deseteral) [inline package-lock.json:1]: Nope - we don't use npm in this project, you have to remove this file.

**Review by Deseteral — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (gorzalczany) [reply README.md:84]: Our authentication is not the same as regular use case described in official documentation. This articles is the best I could find on this topic as it covers our needs perfectly. I have no problem witch changing the link if you have better one. Or we can remove it at all but I think that leaving it may save quite a bit time for potential readers.

Comment (Deseteral) [reply README.md:84]: Let's leave it as it is 👍

**Review by Deseteral — APPROVED** [review]
